src/Languages/ar.js

```diff
@@ -2,6 +2,7 @@ export default {
     'signin': 'تسجيل الدخول',
     'signup': ' مستخدم جديد',
     'topRated': 'اعلى التقيمات',
+    'searchResults': 'سيرتش ريزولتس',
```

Comment:
really?

Reply:
Put it temporarily, forgotten to fix it, whoopsie.